src/main/java/by/itechart/api/controller/UserController.java

```diff
@@ -33,10 +33,15 @@
     }
 
     @PatchMapping("{id}")
-    public ResponseEntity<UserDTO> updateUser(@PathVariable Long id, @RequestBody UserDTO user) {
+    public ResponseEntity<UserDTO> updateUserPartially(@PathVariable Long id, @RequestBody UserDTO user) {
         return new ResponseEntity<>(userService.update(id, user), HttpStatus.OK);
     }
 
+    @PutMapping("{id}")
+    public ResponseEntity<UserDTO> updateUser(@PathVariable Long id, @RequestBody UserDTO userDTO) {
```

Comment:
Why do we need this `PUT` method? Can't we update the whole user using `PATCH`?

Reply:
You are right. But we also discussed that it would be more appropriate to use put for the whole update and patch for the partial update. What do you think?

Reply:
All properties of `User` for the whole update will be available in the `PATCH` method, so what is the difference between these two methods?

Reply:
Technically there is no difference. Okay, I will keep only the patch method.